src/main/webapp/app/exam/participate/exam-participation.component.html

```diff
@@ -163,9 +163,15 @@ <h2 jhiTranslate="artemisApp.examParticipation.submissionSuccessful.title"></h2>
     @if (isAtLeastTutor && !testRunId) {
         <div class="alert alert-warning">
             <h6 jhiTranslate="artemisApp.examParticipation.atLeastTutorStudentExam"></h6>
-            <a [routerLink]="['/course-management', courseId, 'exams', examId]" class="btn btn-primary">
-                <fa-icon [icon]="faGraduationCap" [fixedWidth]="true"></fa-icon>&nbsp;{{ 'artemisApp.examParticipation.goToExamManagement' | artemisTranslate }}
-            </a>
+            @if (isAtLeastInstructor) {
+                <a [routerLink]="['/course-management', courseId, 'exams', examId]" class="btn btn-primary">
+                    <fa-icon [icon]="faGraduationCap" [fixedWidth]="true"></fa-icon>&nbsp;{{ 'artemisApp.examParticipation.goToExamManagement' | artemisTranslate }}
+                </a>
+            } @else {
+                <a [routerLink]="['/course-management', courseId, 'exams']" class="btn btn-primary">
```

Comment:
I think we should still link to this exam directly instead of the dashboard for all exams.
```suggestion
                <a [routerLink]="['/course-management', courseId, 'exams', examId, 'assessment-dashboard']" class="btn btn-primary">
```

Reply:
Assessment dashboard for tutors is available only after exam has ended. Is it a good idea then?

Reply:
Oh, good point. Is there really no other available page specific to the exam? If not, I'm ok with the current solution.

Reply:
Not really. Tutors can only access assessment dashboard. Editors, on the other hand, can also access exercise groups page.